substrate/frame/identity/src/lib.rs

```diff
@@ -213,8 +213,7 @@ pub mod pallet {
 	///
 	/// TWOX-NOTE: OK ― `AccountId` is a secure hash.
 	#[pallet::storage]
-	#[pallet::getter(fn identity)]
-	pub(super) type IdentityOf<T: Config> = StorageMap<
+	pub type IdentityOf<T: Config> = StorageMap<
```

Comment:
CI complaining in pallet-alliance: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/6296254